CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 0.9.0 - 2024-09-03
+- ⚠️ **BREAKING** Add the `BUILDKITE_SPLITTER_RESULT_PATH` required environment variable. See [Migrating to 0.9.0](https://github.com/buildkite/test-splitter/tree/db4cab8cd6c82392553cd80481cf75e3888c2f4c?tab=readme-ov-file#migrating-to-090).
+- Experimental support for Jest by setting `BUILDKITE_SPLITTER_TEST_RUNNER` to `jest`.
+- Update the retry behavior to only retry failed tests rather than files containing test failures.
```

Comment:
```suggestion
- Update the retry behavior to only retry failed tests.
```
This is not accurate, because previously we rely on Rspec internal `--only-failure`. It didn't retry files containing test failures.